charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml

```diff
@@ -89,13 +89,12 @@ webhooks:
       - apiGroups:
           - opentelemetry.io
         apiVersions:
-          - v1alpha1
+          - v1beta1
```

Comment:
The e2e test still uses v1alpha: https://github.com/open-telemetry/opentelemetry-operator/blob/fbbca3d5222f680336a52c21d1442eb87dc1b586/tests/e2e/daemonset-features/01-install.yaml#L1.  Does this change make it impossible to install a v1alpha collector?

Reply:
There is as well a test for v1beta1.

Both versions are supported simultaneously.